config/locales/nl.yml

```diff
@@ -117,6 +117,7 @@ nl:
         admin: Beheerder
         moderator: Moderator
         user: Gebruiker
```

Comment:
Please move this line below [line 120](https://github.com/jeroenpraat/mastodon/blob/cb5527e73002c60fda85514052a0e2c2d9ed68cc/config/locales/nl.yml#L120)